scripts/update-openapi.sh

```
@@ -0,0 +1,50 @@
+
+with open(sys.argv[1], "r") as f:
+    data = json.load(f)
+
+with open(sys.argv[2], "w") as f:
+    json.dump(data, f, indent=2, sort_keys=True, ensure_ascii=False)
+' "$TEMP_FILE" "$OUTPUT_FILE"
+
+# Clean up
+rm "$TEMP_FILE"
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)